LogMonitor/src/LogMonitor/EtwMonitor.cpp

```diff
@@ -768,6 +768,37 @@ std::wstring etwJsonFormat(EtwLogEntry* pLogEntry)
     return oss.str();
 }
 
+///
+/// Format ETW eventlog into a XML output
+///
+std::wstring etwXMLFormat(EtwLogEntry* pLogEntry)
+{
+    std::wostringstream oss;
+
+    // construct the JSON output
+    oss << L"<Source>ETW</Source><LogEntry>";

```

Comment:
nit: I think we mixed this up, the idea was that you leave the format you had proposed, which starts with `<LogEntry>` (as the root tag). This one that we have is not a "valid" XML since it doesn't have a root node/tag.

Reply:
So, the idea is just to take `<LogEntry>` to the beginning.

Reply:
So the change should only be in XML and not JSON?